docs/source/developer/extension_migration.rst

```diff
@@ -0,0 +1,99 @@
+.. _extension_migration:
+
+JupyterLab 1.x to 2.x Extension Migration Guide
+------------------------------------------------
+
+This is a migration guide for updating extensions that support JupyterLab 1.x
+to work in JupyterLab 2.x. We will look at two examples of extensions that
+cover most of the APIs that extension authors might be using:
+
+- ``@jupyterlab/debugger`` migration pull request:
+  https://github.com/jupyterlab/debugger/pull/337/files
+
+- ``@jupyterlab/shortcutui`` migration pull request:
+  https://github.com/jupyterlab/jupyterlab-shortcutui/pull/53/files
+
+Upgrading library versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``@phosphor/*`` libraries that JupyterLab 1.x uses have been renamed to
```

Comment:
Is there a slightly more diplomatic way to say this? phosphor hasn't really been renamed. :-/

Reply:
In the context of a migration guide, it is a pure rename. The history of the library has been discussed elsewhere. Do you find this text problematic?